### DISCUSSION TITLE
Input data improvements / Explore integration of observation data from open repositories

### DISCUSSION THREAD

**Discussion by huitang-earth** [NorESMhub/noresm-land-sites-platform#147] [General]

**Comment by evalieungh** [comment]
Some data comparison is implemented in the analysis notebooks for the Vestland Climate Grid sites using FunCaB data. 

A goal is to use the measured data on e.g. weather/microclimate and plant traits to improve our input data products for the included sites, maybe also to add or customise PFTs for those sites. 

This will take time and involve several projects, so I suggest we transform this issue to a discussion so we can discuss developments and find optimal solutions.

**Comment by evalieungh** [comment]
Hi all, 
I'm in the process of improving the surface data for the VCG ALP1-4 sites with data from SeedClim/FunCaB/INCLINE project observations. I'm adapting @huitang-earth's [script from his Moss PFT work](https://github.com/huitang-earth/MossLichen_testbed/blob/main/scripts/SeedClim_surfacedata_modification.ipynb), with my [modified workflow currently stored in my INCLINE/FATES repo](https://github.com/evalieungh/FATES_INCLINE/blob/main/src/dataprep_surfacedata.ipynb). The finished versions of the data can be uploaded to the LSP as well once I'm done. 

Some questions have come up already:

1. (scientific:) should we change the cover fraction of different PFTs according to observations? It might be difficult to calculate, and will require combining data at different scales (subplot, plot, different years) from different VCG data sources.

2. (technical:) In the [ALP1 data](https://github.com/NorESMhub/noresm-lsp-data/blob/main/sites/ALP1.zip), there are two surface data files, `surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_ALP1_c221011.nc` and `surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_ALP1_c221026.nc`. Is one superfluous? It looks like the other sites only have one surfdata file. Browsing through the two files on Panoply, I have not found any difference between them. @lasseke do you remember something about this?

**Comment by evalieungh** [comment]
Locking this discussion as a duplicate of #166
